week2/week2_day2_exercise.Rmd

```diff
@@ -0,0 +1,205 @@
+#Week 2 _ Day 2 Exercises
+```{r}
+library(tidyverse)
+
+theme_set(theme_bw())
+
+set.seed(42)
+```
+#CHAPTER 4
+#code in book
+```{r}
+library(readr)
+
+ex.1 <- read.csv("ex1.csv")
+summary(ex.1)
+
+pop.1 <- read.csv(file = "pop1.csv")
+summary(pop.1)
+
+sample(pop.1$height,1)
+sample(pop.1$height,1)
+sample(pop.1$height,1)
+
+X <- pop.1$height
+mean(abs(X - 170) <= 10)
+
+Y <- c(6.3, 6.9, 6.6, 3.4, 5.5, 4.3, 6.5, 4.7, 6.1, 5.3)
+abs(Y - 5)
+abs(Y - 5) <= 1
+mean(abs(Y - 5) <= 1)
+```
+
+#Question 4.1.1 - What is the value of p?
+Answer - The probabilities of the random variables of Y should add up to 1, so:
+p + 2p + 3p + 4p + 5p + 6p = 21p 
+21p = 1
+p = 1/21
+
```

Comment:
In Rmd you can put $ around your math to make it appear better when you eventually export it.

$p + 2p$ Github has it too! You may already know LaTeX, it follows the same syntax. (roughly)